README.md

```diff
@@ -14,12 +14,13 @@
 It is designed to build applications that run on desktop and mobile devices with a
 single codebase.
 
-Version 1.4 is the current release - it introduced high performance collection widgets,
-Card, Separator and FileIcon widgets as well as a folder open dialog.
-It also saw a theme refresh updating the colors and button styles for a more
-material design look.
-We are now working towards [2.0](https://github.com/fyne-io/fyne/milestone/6)
-which aims to add data bindings, animations and more!
+Version 2.0 is the current release of the Fyne API, this represented the first release since
+1.0 that may break some API usage. It also added new features including data binding, animation,
+storage repositories and a new more flexible theme API.
+We also refreshed the default theme, adding animations, a focus colour and 
```

Comment:
Should the README read in present or past tense?

L20-21 read in past tense.

The rest of the file reads in present tense,

Until L22-23 which read in present future tense.

Reply:
Well it sort of depends. Mostly present tense works.
I used past to indicate (and this is the first time) that something has been changed that implies developers must change their code.
Maybe the future tense portions are not needed but it seemed right to indicate where a developer could choose to perform an action.

Reply:
Totally fair! Once this change in tense was intentional, it makes sense.